enterprise/views.py

```diff
@@ -1352,12 +1356,16 @@ def post(self, request, enterprise_uuid, course_id):
         """
         Process a submitted track selection form for the enterprise.
         """
-        # We need to override this constant here, because it's calculated at load time and there's no
+        # We need to override these constants here, because they are calculated at load time and there's no
         # Site context at load time.
         LMS_COURSEWARE_URL = urljoin(
             get_configuration_value('LMS_ROOT_URL', settings.LMS_ROOT_URL),
             '/courses/{course_id}/courseware',
         )
+        LMS_START_PREMIUM_COURSE_FLOW_URL = urljoin(
+            get_configuration_value('LMS_ROOT_URL', settings.LMS_ROOT_URL),
+            '/verify_student/start-flow/{course_id}/',
```

Comment:
You are getting the value twice. @Squirrel18 
```suggestion
        # Site context at load time.
        lms_root_url = get_configuration_value('LMS_ROOT_URL', settings.LMS_ROOT_URL)
        LMS_COURSEWARE_URL = urljoin(
            lms_root_url,
            '/courses/{course_id}/courseware',
        )
        LMS_START_PREMIUM_COURSE_FLOW_URL = urljoin(
            lms_root_url,
            '/verify_student/start-flow/{course_id}/',
```